### ISSUE TITLE
Adding music folder renders deadbeef window unusable while scanning

### ISSUE THREAD

**Issue by Oleksiy-Yakovenko** [DeaDBeeF-Player/deadbeef#690]
Original [issue 788](https://code.google.com/p/ddb/issues/detail?id=788) created by Alexey-Yakovenko on 2012-04-26T21:45:13.000Z:

<b>What steps will reproduce the problem?</b>
<b>Какие шаги приводят к воспроизведению</b>
<b>проблемы?</b>
1. Add a music folder to a playlist with more than a small amount of music.
2. Deadbeef window becomes unusable due to scanning window on top.
<b>3.</b>

<b>What is the expected output? What do you see instead?</b>
<b>Какой ожидаемый вывод? Что вы видите</b>
<b>вместо него?</b>
  If you do not have hotkeys/shortcuts set up you cannot play/pause/skip/stop/etc. until the scanning is complete, or you cancel it.

<b>What version of the product are you using? On what operating system?</b>
<b>Какую версию продукта вы используете? На</b>
<b>какой операционной системе?</b>
  0.5.2-1, Arch Linux.

<b>Please provide any additional information below.</b>
<b>Пожалуйста, предоставьте любую</b>
<b>дополнительную информацию ниже</b>

**Comment by cocoonkid** [comment]
Yes, duplicate of #2605 . I imported about 1400 tracks. Arch Linux as well.
